<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Definitions</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Values</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Explicit Values</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="definitionsTreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QTreeView" name="ctx2TreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="3">
     <widget class="QTreeView" name="valuesTreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="3">
     <widget class="QTreeView" name="explicitValuesTreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Environment - Context 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTreeView" name="environmentTreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QTreeView" name="ctx3TreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Context 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Context 3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_variable_definition">
   <property name="text">
    <string>Add variable definition</string>
   </property>
  </action>
  <action name="actionAdd_definition_group">
   <property name="text">
    <string>Add definition group</string>
   </property>
  </action>
  <action name="actionReload_environment">
   <property name="text">
    <string>Reload environment</string>
   </property>
  </action>
  <action name="actionAdd_variable_to_context_2">
   <property name="text">
    <string>Add variable to context</string>
   </property>
  </action>
  <action name="actionAdd_variable_to_context_3">
   <property name="text">
    <string>Add variable to context</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
